src/components/utils.ts

```diff
@@ -607,6 +607,31 @@ export function getValidUrl(url: string): string {
   }
 }
 
+/**
+ * Creates a nanoid
+ * 
+ * @param t Number to fill array
+ * @returns {string} Id with t characters
+ */
+function nanoid(t = 21): string {
```

Comment:
@neSpecc 
Would you like to use the original npm library?
https://www.npmjs.com/package/nanoid